_config/dirs.txt

```diff
@@ -1,2 +1,3 @@
-/rfanfs/pnl-zorro/
-tb571@eris1n3.research.partners.org:/data/pnl
+/data/pnl/
+/data/pnlx/
+/data/predict1/
```

Comment:
Colin, I want you to undo this commit. It was right before. This `dirs.txt` serves as an example and so we want it to contain both rfanfs and data.

Reply:
I undid the commit, but I am keeping dirs.txt in .gitignore to prevent issues when testing.